Oceanwater 180 GUI for plexil plan selection (default perspective file)

Keeganfn opened the pull request:
Added PlexilPlanSelection plugin to the default.perpective file for rqt. This will automatically bring up the plugin alongside the others. Should only be merged after OCEANWATER-180 PR on ow_autonomy is approved and merged.


## Linked Issues:
| Jira Ticket 🎟️ | [OCEANWATER-180](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-180) |
| :----------- | :----------- |
| Github :octocat:  | https://github.com/nasa/ow_autonomy/pull/52 |


## Summary of Changes
* Modified default.perspective file for rqt window. Now will automatically bring up the new PlexilPlanSelection plugin alongside the other existing plugins. Should only be merged after OCEANWATER-180 PR on ow_autonomy is approved and merged.

OCEANWATER-180 GUI for plexil plan selection: https://github.com/nasa/ow_autonomy/pull/52



## Test
* After OCEANWATER-180 has been merged to ow_autonomy melodic-devel branch, run ```roslaunch ow europa_terminator_workspace.launch``` or any other launch file that uses the rqt default.perspective file at launch. 

If the PlexilPlanSelection plugin shows up alongside the other plugins in the rqt window then everything works.

Samahu reviewed (changes requested):
- [ ] I think we better keep the "Control and Viz" as the default tab.
- [ ] Consider adding the "PlexilPlanSelection" plugin to the right of the three pre-existing ones.

Keeganfn commented:
Per @Samahu suggestion I changed the order of the plugins so that PlexilPlanSelection is now the last tab instead of the first, and moved the control and viz back to the default tab.

kmdalal reviewed (approved):
Looks good!
